app/project/[id]/issue/[issueId]/page.tsx

```diff
@@ -44,6 +46,10 @@ export default function viewIssuePage({ params }: {
     const [canControl, setCanControl] = useState(false);
     const [project, setProject] = useState<Project>();
     const [issue, setIssue] = useState<Issue>();
+    const [canApply, setCanApply] = useState(false);
+    const { data: session } = useSession();
```

Comment:
There's multiple of the same variable, this creates an error that doesn't let you compile.
```
Error: 
  × the name `session` is defined multiple times
    ╭─[C:\Users\karol\Documents\GitHub\IT-sponsor\app\project\[id]\issue\[issueId]\page.tsx:42:1]
 42 │ export default function viewIssuePage({ params }: {
 43 │     params: { id: number, issueId: number }
 44 │ }) {
 45 │     const { data: session } = useSession();
    ·                   ───┬───
    ·                      ╰── previous definition of `session` here
 46 │     const [canControl, setCanControl] = useState(false);
 47 │     const [project, setProject] = useState<Project>();
 48 │     const [issue, setIssue] = useState<Issue>();
 49 │     const [canApply, setCanApply] = useState(false);
 50 │     const { data: session } = useSession();
    ·                   ───┬───
    ·                      ╰── `session` redefined here
 51 │     const [hasAccess, setHasAccess] = useState(false);
 52 │     const [isAssigned, setIsAssigned] = useState(false);
 52 │     const [loading, setLoading] = useState(true);

    ╰────
 ```

Reply:
Fixed